src/votekit/elections/election_types.py

```diff
@@ -951,3 +951,117 @@ def __init__(
         super().__init__(profile, ballot_ties)
         self.seats = seats
         self.tiebreak = tiebreak
+
+class HighestScore(Election):
+    """
+    Conducts an election based on points from score vector. 
+    Chooses the m candidates with highest scores.
+    Ties are broken by randomly permuting the tied candidates.
+
+    **Attributes**
+
+    `profile`
+    :   PreferenceProfile to run election on.
+
+    `seats`
+    :   number of seats to be elected
+
+    `score_vector`
+    : list of floats where ith entry denotes the number of points given to candidates
+        ranked in position i.
+
+    `tiebreak` (optional)
+    : tiebreak method, defaults to random, supports 'none', 'random',
+
+    `ballot_ties` (optional)
+    : resolves ties in ballots. Should only be set to True if you want ballots
+        to have full linear rankings.
```

Comment:
Would it be better for this to accept a method to make it more flexible for users?

Reply:
If a user wanted to apply their own ballot tie resolving method, they could pass it to the existing `clean_profile` function before passing the profile into the election class, so I think we can leave the `ballot_ties` as is!

I do think it would be nice for users to be able to pass in their own tiebreak method, so I'll make that change.